astroquery/esa/hsa/tests/setup_package.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
There are no data files for this module, so no need for this file either.

Reply:
Removed.